tests/ui/deriving/smart-pointer-bounds-issue-127647.rs

```diff
@@ -0,0 +1,78 @@
+//@ check-pass
```

Comment:
Please add a `-Zunpretty=expanded` test, I'd really like to see what the expanded output looks like.

Reply:
The expansion test is added, too.